### ISSUE TITLE
Question: is the project on hold?

### ISSUE THREAD

**Issue by yuvalr1** [netdur/llama_cpp_dart#42]
Quote from readme:
> This project, a Dart binding for llama.cpp, is currently on hold as we await the porting of llama.cpp helpers code to C

Is there a link, potentially to the github issue in the [llama.cpp repository](https://github.com/ggerganov/llama.cpp) so we can understand and follow on the status of the dependency?

Thank you very much for this project!

**Comment by netdur** [comment]
@yuvalr1 yes, but not for long as https://github.com/ggerganov/llama.cpp/issues/5214 was resolved lately, this will allow generate binding without C wrappers.

**Comment by yuvalr1** [comment]
Thanks @netdur!
I'm ok with closing the issue unless you want to leave it open.

**Comment by netdur** [comment]
@ArachnaDigitalLtd dev branch is now available for review. while it's still a work in progress, the implementation is functional and ready for testing
@yuvalr1 I will close once with release of v0.0.8

**Comment by JulienElkaim** [comment]
Congrats for resuming work on this project @netdur  ! 
Highly beneficial for the Flutter community. I will give it a try as I wish to move from my prev implementation to yours. Hope to contribute one day if I get to use it.

Is there any details on what is actually supported from the official current llamacpp API / what is not?

Thank you again

**Comment by guyluz11** [comment]
> I will close once with release of v0.0.8

It got released, I think this issue can be closed.

**Comment by netdur** [comment]
No — actively shipping. The "on hold" note is long gone from the README. The 0.9.x rewrite is the current line; latest is `0.9.0-dev.6` on pub.dev today.

Closing.
